webofneeds/won-owner-webapp/src/main/webapp/app/components/feedback-grid.js

```diff
@@ -70,6 +70,7 @@ function genComponentConf() {
                     console.log("RATE BAD");
                     this.connections__close(this.connectionUri);
                     this.connections__rate(this.connectionUri, won.WON.binaryRatingBad);
+                    this.router__stateGoCurrent({connectionUri: null})
```

Comment:
Maybe this route change should be dispatched in the connections__close action if the current route matches the connection route?